dev/docker/ubuntu_node.mingw.dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:22.04 AS base
```

Comment:
Please add this docker file's name to `github/workflows/ci.yml` so that it is tested.